master/internal/api_project.go

```diff
@@ -185,6 +190,83 @@ func (a *apiServer) getProjectColumnsByID(
 	if err != nil {
 		return nil, err
 	}
+
+	trialIDs := make([]int, 0, len(hyperparameters)+1)
+	// dummy value so bun doesn't generate invalid sql
+	trialIDs = append(trialIDs, -1)
+	for _, hparam := range hyperparameters {
+		if hparam.BestTrialID != nil {
+			trialIDs = append(trialIDs, *hparam.BestTrialID)
+		}
+	}
+	summaryMetrics := []struct {
+		ID             int
+		SummaryMetrics model.JSONObj
+	}{}
+
+	trialsQuery := db.Bun().NewSelect().
+		Column("id").
+		Column("summary_metrics").
```

Comment:
nitpick: this should also work, and ends up being a bit more terse.

```suggestion
		Column("id", "summary_metrics").
```